ql/src/test/queries/clientpositive/acid_direct_delete.q

```diff
@@ -0,0 +1,18 @@
+set hive.mapred.mode=nonstrict;
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.acid.direct.insert.enabled=true;
+
+drop table if exists full_acid;
+
+set mapreduce.job.reduces=7;
+
+create external table ext(a int) stored as textfile;
+insert into table ext values(1),(2),(3),(4),(5),(6),(7), (8), (9), (12);
+create table full_acid(a int) stored as orc tblproperties("transactional"="true");
+
+insert into table full_acid select * from ext where a != 3 and a <=7 group by a;
```

Comment:
why do you need `group by`?

Reply:
there is nothing to group, `a` is already distinct. please check the complete test snippet I've shared here.

Reply:
I can repro the issue using the test snippet `acid_direct_update_delete.q ` .  I have rewritten and updated.
In the test steps provided in JIRA , this is not reproducible without group by though.

Reply:
I was just wondering why without group by the new test case was not reproducing the issue. We are selecting it from a text file and we needed to group by to distribute. 

And Yes, we don't need `group by ` in the `acid_direct_update_delete.q ` , so I have not added it. 
I mentioned I have rewritten in the sense that I have written the test case in the existing file and deleted the new one.

Reply:
test_update_bucketed is bucketed, so it's already distributed